Assets/Scripts/PlayerMovement.cs

```diff
@@ -0,0 +1,63 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PlayerMovement : MonoBehaviour
+{
+    public float moveSpeed = 1.0f; // Should Always Be Positive
+    public float jumpForce = 1.0f; // Should Always Be Positive
+
+    private float xInput;
+    private float yInput;
+
```

Comment:
Lots of weird spacing in this file... maybe there's a code formatter for Unity?

(Unity convention could be different from Python though, so I might be nitpicking)

Reply:
I was making space to separate out the variables that are public and could be edited in the unity editor and the private ones that are only used in this file.

Reply:
And on conventions I haven't seen nor know of any Unity Conventions